tox.ini

```diff
@@ -1,16 +1,32 @@
 [tox]
-envlist = py27-django{18,19}, flake8
+envlist = py{27,34,35,36}-django{18,19,110,111}, flake8
 
 [flake8]
+ignore = F403
```

Comment:
I think we should remove this in favor of inline skipping of this specific error.

Reply:
:thumbsup: